### ISSUE TITLE
[a11y]: Product left nav panel interfering with walkme tour

### ISSUE THREAD

**Issue by bhaskarbhattarai-ibm** [carbon-design-system/carbon#19701]
### Package

carbon-components

### Package version

Common Header version 0.1.28

### Automated testing tool and ruleset

manual testing

### Description

Hi Team, 

In the IBM webMethods MFT and B2B product UI, we have a carbon component left nav that is expandable and collapsible. This interferes with the user experience during the WalkMe tours because the WalkMe elements need to adapt every time with the left nav components collapsing and expanding. Here is a video of the issue, please take a look at it and suggest a workaround.

Please let me know for additional information.

### Reproduction/example

Not sure about this

### Steps to reproduce

I am not able to attach the video. Please reach out to me and I can give you a demo of the issue we are facing.

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by Gururajj77** [comment]
Hey @bhaskarbhattarai-ibm , could you recreate this issue in the [stackblitz](https://github.com/carbon-design-system/sandboxes/blob/main/README.md)?

**Comment by bhaskarbhattarai-ibm** [comment]
Hi @Gururajj77 ,

Unfortunately I am not familiar with stackblitz to repro the issue. Will it help if I demo you the issue? I can setup a quick screen sharing session.

Thanks,
Bhaskar

**Comment by tay1orjones** [comment]
@bhaskarbhattarai-ibm we can't know what action to take without a clear reproduction for us to debug. You can share your screen recording in the internal `#carbon-react` slack channel and link it here if you prefer.

**Comment by 2nikhiltom** [comment]
Hey @bhaskarbhattarai-ibm : are you able to provide a reproduction for us to debug?

**Comment by heloiselui** [comment]
Thank you for opening this issue. We're closing it because we couldn't reproduce the problem. If it keeps happening, feel free to open the issue again and show us how to reproduce it.
